File: src/codes/Users_Manual/modulation/csv_0.py
~import pathlib
~from pyautd3 import Hz
from pyautd3.modulation.audio_file import Csv, CsvOption

Csv(
    path=pathlib.Path("path/to/foo.csv"),
    sampling_config=4000 * Hz,
    option=CsvOption(
        delimiter=",",
    ),
)

File: src/codes/Users_Manual/modulation/csv_0.cs
using AUTD3Sharp.Modulation.AudioFile;

~using static AUTD3Sharp.Units;
new Csv(
    path: "path/to/foo.csv",
    samplingConfig: 4000f * Hz,
    option: new CsvOption
    {
        Delimiter = ',',
    }
);